src/Data/Config.php

```diff
@@ -51,6 +52,96 @@ public function __construct() {
 			'graphql_wp_query_cursor_pagination_stability'
 		], 10, 2 );
 
+		if ( ! defined( 'ABSPATH' ) ) {
+			exit;
+		}
+		
+		/**
+		 * Copied from https://github.com/wp-graphql/wp-graphql/issues/274#issuecomment-510150571
+		 * Shoutouts to epeli!
+		 * 
+		 * Add missing filters to WP_User_Query class.
+		 */
+		add_filter( 'pre_user_query', function ( $query ) {
+		
+				if ( ! $query->get( 'suppress_filters' ) ) {
+						$query->set( 'suppress_filters', 0 );
+				}
+		
+				if ( ! $query->get( 'suppress_filters' ) ) {
+		
+						/**
+						 * Filters the FROM clause of the query.
+						 *
+						 * Specifically for manipulating paging queries.
+						 **
+						*
+						* @param string $where The FROM clause of the query.
+						* @param WP_User_Query $query The WP_User_Query instance (passed by reference).
+						*/
+						$query->query_from = apply_filters_ref_array( 'users_from', array( $query->query_from, &$query ) );
+		
+						/**
+						 * Filters the WHERE clause of the query.
+						 *
+						 * Specifically for manipulating paging queries.
+						 **
+						*
+						* @param string $where The WHERE clause of the query.
+						* @param WP_User_Query $query The WP_User_Query instance (passed by reference).
+						*/
+						$query->query_where = apply_filters_ref_array( 'users_where', array( $query->query_where, &$query ) );
```

Comment:
I think this should be named `graphql_users_where` to avoid collisions if somebody is doing the same thing in userland.

Also the other `apply_filters_ref_array`s could be removed since they are not used.